internal/testutil/pkgdata.go

```diff
@@ -13,150 +12,31 @@ import (
 
 var PackageData = map[string][]byte{}
 
-func init() {
-	baseFilesData, err := base64.StdEncoding.DecodeString(baseFilesBase64)
-	if err != nil {
-		panic("broken package base64")
-	}
-	PackageData["base-files"] = baseFilesData
+var testPackageEntries = []TarEntry{
+	Dir(0755, "./"),
+	Dir(0755, "./dir/"),
+	Reg(0644, "./dir/file", "12u3q0wej	ajsd"),
+	Reg(0644, "./dir/other_file", "kasjdf0"),
+	Dir(0755, "./dir/nested/"),
+	Reg(0644, "./dir/nested/file", "0jqei"),
+	Reg(0644, "./dir/nested/other_file", "1"),
+	Dir(0755, "./dir/several/levels/deep/"),
+	Reg(0644, "./dir/several/levels/deep/file", "129i381		"),
+	Dir(0755, "./other_dir/"),
+	Dir(01777, "./parent/"),
+	Dir(0764, "./parent/permissions/"),
+	Reg(0755, "./parent/permissions/file", "ajse0"),
```

Comment:
nit: what do you think about having the usual path structure e.g.

```
./etc/
./etc/foo.conf
./etc/bar.conf
./etc/baz/
./etc/baz/baz.conf
./tmp/
./usr/
./usr/bin/
./usr/bin/foo
```

Reply:
This is a good idea and I did the same thing in previous iterations. However, in discussions with Gustavo we agreed that the best thing would be either to have a real package or test data where the test data clearly indicates its purpose (e.g. no more /usr/bin/hello). We agreed that the latter was better.